roles/ceph-common/defaults/main.yml

```diff
@@ -20,6 +20,7 @@ ceph:
     - ntp
     - hdparm
     - libvirt-bin
+    - acl
```

Comment:
This is already [included and defined in the `common` role](https://github.com/blueboxgroup/ursula/blob/25f310f714ec313018ace93ea2a1d17859b6700b/roles/common/tasks/system-tools.yml#L9)